background/services/chain/tests/index.unit.test.ts

```diff
@@ -27,6 +29,30 @@ describe("Chain Service", () => {
     chainService = await createChainService()
   })
 
+  describe("Alarms", () => {
+    describe("handleBlockPricesAlarm", () => {
+      it("should only poll for block prices on active networks", async () => {
+        const sixMinsAgo = Date.now() - 6 * MINUTE
+        ;(
+          chainService as unknown as ChainServiceExternalized
+        ).lastUserActivityOnNetwork = {
+          [ETHEREUM.chainID]: Date.now(),
+          [POLYGON.chainID]: sixMinsAgo,
+          [OPTIMISM.chainID]: sixMinsAgo,
+        }
```

Comment:
```suggestion
        const ChainServiceExternalized =
          chainService as unknown as ChainServiceExternalized

        ChainServiceExternalized.lastUserActivityOnNetwork = {
          [ETHEREUM.chainID]: Date.now(),
          [POLYGON.chainID]: sixMinsAgo,
          [OPTIMISM.chainID]: sixMinsAgo,
        }
        ;[ETHEREUM, POLYGON, OPTIMISM].forEach((network: EVMNetwork) => {
          ChainServiceExternalized.subscribedNetworks.push({
            network,
            provider: {} as unknown as SerialFallbackProvider,
          })
        })
```

This breakds the indexing/tests/index.unit.test.ts` with `IndexingService › service start › should update cache once token lists load`. Probably there is some state persisting in the mocks. Didn't dig to the end.